packages/desktop-client/src/components/payees/ManagePayeesWithData.tsx

```diff
@@ -67,7 +67,6 @@ export function ManagePayeesWithData({
         return;
       }
 
-      await dispatch(getPayees());
```

Comment:
No need for this since this is already handles by the global sync-event listener